lib/ansible/modules/network/fortimanager/fmgr_secprof_av.py

```diff
@@ -0,0 +1,1683 @@
+      - List of multiple child objects to be added. Expects a list of dictionaries.
+      - Dictionaries must use FortiManager API parameters, not the ansible ones listed below.
+      - If submitted, all other prefixed sub-parameters ARE IGNORED.
+      - This object is MUTUALLY EXCLUSIVE with its options.
+      - We expect that you know what you are doing with these list parameters, and are leveraging the JSON API Guide.
+      - WHEN IN DOUBT, USE THE SUB OPTIONS BELOW INSTEAD TO CREATE OBJECTS WITH MULTIPLE TASKS
+    required: false
+
+  smtp_archive_block:
+    description:
+      - Select the archive types to block.
+      - FLAG Based Options. Specify multiple in list form.
+      - flag | encrypted | Block encrypted archives.
+      - flag | corrupted | Block corrupted archives.
+      - flag | multipart | Block multipart archives.
+      - flag | nested | Block nested archives.
+      - flag | mailbomb | Block mail bomb archives.
+      - flag | unhandled | Block archives that FortiOS cannot open.
+      - flag | partiallycorrupted | Block partially corrupted archives.
+      - flag | fileslimit | Block exceeded archive files limit.
+      - flag | timeout | Block scan timeout.
+    required: false
+    choices:
+      - encrypted
+      - corrupted
+      - multipart
+      - nested
+      - mailbomb
+      - unhandled
+      - partiallycorrupted
+      - fileslimit
+      - timeout
+
+  smtp_archive_log:
+    description:
+      - Select the archive types to log.
+      - FLAG Based Options. Specify multiple in list form.
+      - flag | encrypted | Log encrypted archives.
+      - flag | corrupted | Log corrupted archives.
+      - flag | multipart | Log multipart archives.
+      - flag | nested | Log nested archives.
+      - flag | mailbomb | Log mail bomb archives.
+      - flag | unhandled | Log archives that FortiOS cannot open.
+      - flag | partiallycorrupted | Log partially corrupted archives.
+      - flag | fileslimit | Log exceeded archive files limit.
+      - flag | timeout | Log scan timeout.
+    required: false
+    choices:
+      - encrypted
+      - corrupted
+      - multipart
+      - nested
+      - mailbomb
+      - unhandled
+      - partiallycorrupted
+      - fileslimit
+      - timeout
+
+  smtp_content_disarm:
+    description:
+      - Enable Content Disarm and Reconstruction for this protocol.
+      - choice | disable | Disable Content Disarm and Reconstruction for this protocol.
+      - choice | enable | Enable Content Disarm and Reconstruction for this protocol.
+    required: false
+    choices: ["disable", "enable"]
+
+  smtp_emulator:
+    description:
+      - Enable/disable the virus emulator.
+      - choice | disable | Disable the virus emulator.
+      - choice | enable | Enable the virus emulator.
+    required: false
+    choices: ["disable", "enable"]
+
+  smtp_executables:
+    description:
+      - Treat Windows executable files as viruses for the purpose of blocking or monitoring.
```

Comment:
I've seen this a couple of times at least now - I'm unclear if this should really be saying Windows executable, or if that's a copy/paste issue and it should say SMTP executable.  If it shouldn't say windows, search for this because I know I saw it before this comment...